src/main/java/com/google/devtools/build/lib/remote/util/DigestUtil.java

```diff
@@ -52,14 +56,14 @@ private ActionKey(Digest digest) {
     }
   }
 
-  private final DigestHashFunction hashFn;
```

Comment:
Ah, sorry, that conflicts with a patch I have in flight. I want to be able to map from this hash function to the corresponding enum value in https://github.com/bazelbuild/remote-apis/blob/master/build/bazel/remote/execution/v2/remote_execution.proto#L1281
So I do need the DigestHashFunction wrapper, because it contains the names which I can use to translate (and in the next patch, I will add the enum value directly to it as well).